### ISSUE TITLE
rotate topos on mt alexander

### ISSUE THREAD

**Issue by scd** [theCrag/website#2165]
http://www.thecrag.com/climbing/australia/mt-alexander/area/421027890

**Comment by brendanheywood** [comment]
I'm increasingly uncomfortable fixing these manually, we don't know how many there are and I suspect there could be hundreds. We need to find the root cause and fix them all in a batch

**Comment by scd** [comment]
I have just done a random review of 400 topos and only found it came up in one other place

http://www.thecrag.com/climbing/australia/mt-alexander/area/421020291

It looks like the problem is rare but when it happens there may be a group of bad topos. I don't think it is hundreds, but it might be dozens.

I am going to do a review of all topos, it may only take a couple of hours, then at least we will know the size of the problem.

**Comment by brendanheywood** [comment]
What about photos vs topos?

**Comment by scd** [comment]
I am not to concerned about photos. Rotate button on the ui would be fine
for photos.

There could be multiple issues and topos are critical, so regardless we
need to audit topos

**Comment by scd** [comment]
see also #2148

**Comment by scd** [comment]
upto here in audit

http://www.thecrag.com/climbing/world/topos?page=61

**Comment by scd** [comment]
Audit complete and topos corrected. I went quickly through the topos so there is a chance I missed one or two.

There were about 7 areas where one or more topo images were rotated relative to their dimensions and lines. 

From memory I think we changed logic from the rotation database setting being an exif override to a rotation. It should not make any difference unless the camera exif is set wrong, which I know happens from time to time.

Closing.
